src/main/java/noobconversation/Noob.java

```diff
@@ -0,0 +1,8 @@
+package noobconversation;
+
+public class Noob {
+    public static void main(String[] args) {
+        Conversation a = new Conversation();
+        a.startConversationInitiateLogo();
```

Comment:
Like I mentioned before, you can make the code more obvious by having 3 calls:
`printWelcomeMessage()`
`startConversation()`
`printGoodbyeMessage()`